### ISSUE TITLE
Progress bars missing in 0.95.X and 1.2.1

### ISSUE THREAD

**Issue by jjlawren** [kalkih/mini-media-player#138]
Entities which used to display progress bars no longer display them, seemingly with the upgrade to 0.95.X. Verified the progress bars still work fine on the same entities with the built-in `media-control` card.

**Comment by kalkih** [comment]
Hmm okay, I did manage to recreate the issue, but only when/if the card had cover artwork, will look into it, thanks! :+1:

**Comment by YamiFrankc** [comment]
I'm having this problem too. I just installed it today using haccs.
The progress bar is displayed only when media us paused, but not while playing.
```

      - artwork: full-cover
        entity: media_player.cuarto_frank_display
        group: true
        hide:
          progress: false
          runtime: false
        icon: 'mdi:music'
        type: 'custom:mini-media-player'
```

**Comment by kalkih** [comment]
Okay, should be fixed with 34c62c9, can probably roll it out in a new release tonight, thanks for your patience!

**Comment by kalkih** [comment]
New release now available, should be fixed!
